pkg/kanx/kanx.proto

```diff
@@ -5,10 +5,11 @@ option go_package = "github.com/kanisterio/kanister/pkg/kanx";
 package kanx;
 
 service ProcessService {
-  rpc CreateProcesses (CreateProcessRequest) returns (Process) {}
+  rpc CreateProcess (CreateProcessRequest) returns (Process) {}
+  rpc GetProcess (ProcessPidRequest) returns (Process) {}
   rpc ListProcesses (ListProcessesRequest) returns (stream Process) {}
-  rpc Stdout (ProcessOutputRequest) returns (stream Output) {}
-  rpc Stderr (ProcessOutputRequest) returns (stream Output) {}
+  rpc Stdout (ProcessPidRequest) returns (stream Output) {}
```

Comment:
Could you please clarify this Change?

Reply:
@tdmanv Ah.  There are RPC requests in the protocol that take only one argument, the PID.  I've simplified these RPC requests by consolidating the `Process*Request` to `ProcessPidRequest`

Reply:
Ah, I understand, thanks